src/services/validation.js

```diff
@@ -1,36 +1,32 @@
 import firebaseAPI from "@/services/API/checkUser.js";
-import { sendPlio } from "@/services/sendPlio";
-
-//this function is invoked only for the check of SRN's.
-export async function validateSRN(userID, validateCount, isSingleEntryOnly, redirectID, doesUserExist){
 
+// this function is invoked only for the check of SRN's.
+// validateCount = 0, when the user gets validated on the first try
+// validateCount = 1, when the user fails the first check
+// validateCount = 2, when user fails the second check as well -> letting the user go through irrespective.
+export async function validateSRN(userID, validateCount, isUserValid){
     var invalidLoginMessage = ""
-    //checks the basic conditions of SRN: 
-    //      - starts with only '1'
+
+    //checks the basic condition of SRN: 
     //      - does not contain same number patterns like '111...', '222...', etc.
-    if(!(/^([0-9])\1*$/.test(userID)) && userID.toString()[0] == '1'){
+    if(!(/^([0-9])\1*$/.test(userID))){
 
         //if the basic conditions are satisified, only then the request is sent for further checking
-        doesUserExist = await firebaseAPI.checkUserExists(userID);
+        isUserValid = await firebaseAPI.checkUserExists(userID);
```

Comment:
you can just return here as well right?
If the user is valid, then you can just return with that object here. 
This way, going forward, you won't need to check `!isUserValid` again and again.

Reply:
won't there be two return statements that way?

Reply:
That isn't wrong na?
I don't think there's any problem with having multiple return statements!